src/pages/components/Overlay.qml

```diff
@@ -158,6 +158,29 @@ PanelBackground {
         source: "image://theme/graphic-gradient-edge"
     }
 
+    Label {
+        id: privateModeLabel
```

Comment:
The label could be also at the same place as the PrivateModeTexture should be, see below.

@ BrowserPage.qml after the MouseArea ~133

```
        Browser.PrivateModeTexture {
            anchors.fill: parent
            active: !overlay.toolBar.visible && webView.privateMode

            Label {
                id: privateModeLabel
                x: Theme.paddingLarge
                // Allow only half of the width
                width: parent.width / 2
                truncationMode: TruncationMode.Fade
                anchors {
                    bottom: parent.bottom
                    bottomMargin: (overlay.toolBar.toolsHeight - height) / 2
                    horizontalCenter: parent.horizontalCenter
                }

                //: Label for private browsing above address bar
                //% "Private browsing"
                text: "Private browsing"
                color: Theme.highlightColor
                font.pixelSize: Theme.fontSizeLarge
            }
        }
    }
```